internal/impl/publisher/request_reply_message_publisher_impl.go

```diff
@@ -877,12 +877,12 @@ func (publisher *requestReplyMessagePublisherImpl) handleReplyMessage(msgP core.
 	defer publisher.rxLock.Unlock()
 	corEntry, ok := publisher.requestCorrelationMap[correlationID]
 	if !ok {
-		publisher.logger.Debug(fmt.Sprintf("Received reply message[%p] with correlationID[%s] without correlation entry for publisher", msgP, correlationID))
+		publisher.logger.Debug(fmt.Sprintf("Received reply message[0x%x] with correlationID[%s] without correlation entry for publisher", msgP, correlationID))
 		return false
 	}
 	if corEntry.received {
 		// return false to return the message
-		publisher.logger.Debug(fmt.Sprintf("Received reply message[%p] with correlationID[%s] that already has response", msgP, correlationID))
+		publisher.logger.Debug(fmt.Sprintf("Received reply message[0x%x] with correlationID[%s] that already has response", msgP, correlationID))
```

Comment:
@TrentDaniel, please why did you change the Sprintf formatting verb from `%p` to `%x` ?

Reply:
@oodigie , Ragnar is correct, the formatting verb no longer matched the type. I could've instead done some conversions to get a pointer just for the print statement, but that would have require multiple casts and new objects to be created. Since it was only two lines that needed to be printed, I just formatted the integer in hexadecimal, and prefixed it with a hardcoded string of `0x` so that it would appear as before. The prefix is required because while the `%p` verb will automatically prepend that prefix to the hex repr of the int, the `%x` will not. This can be observed through this example:  https://go.dev/play/p/7v9BwWInXIn.